Teatoller/tutorial
# #162848652 Project Set-Up

#### What does this PR do?
* This PR creates the project shell and file structure.

#### Description of Task to be completed?
* Create a repository in Github.
* Clone the repository in the local computer.
* Create a virtual environment.
* Install dependencies.
* Create project directory.
* Create a single App "quickstart".

#### How should this be manually tested?

* ``` $ mkdir project```
* ```$ git clone 'https://github.com/Teatoller/tutorial.git'```
* ``` $cd tutorial```
* ```$ python3 -m pip install --user --upgrade pip```
* ```$ python3 -m pip install --user virtualenv```
* ```$ python3 -m virtualenv env```
* ```$ source env/bin/activate```
* ```$ pip install -r requirements.txt```
* ```$ python3 manage.py runserver```
* Paste the url in browser : http://127.0.0.1:8000/

You should be able to see the screenshot captured below in your browser.

#### Any background context you want to provide?

* Note to Deactivate the virtual environment use ```$ deactivate``` 

#### What are the relevant pivotal tracker stories?
* https://www.pivotaltracker.com/n/projects/2234238
* id#162848652

#### Screenshots (if appropriate)

* Application Shell Succesfully Activated.
<img width="1280" alt="screenshot 2018-12-26 at 21 27 31" src="https://user-images.githubusercontent.com/39020475/50453907-29394500-0955-11e9-9ab3-65e9a065a0d6.png">

#### Questions:
* n/a
